### DISCUSSION TITLE
Ability to export a CSV from user list

### DISCUSSION THREAD

**Discussion by buehler** [zitadel/zitadel#2220] [Ideas]
When an organization has a big load of users (3280 in our case :) it would be helpful to export the user list to filter them according to some criteria.

Some "simple" csv export with the typical user data (meaning name, email, status, phone, whatever user data.) would be cool.

**Comment by livio-a** [comment]
This would definitely be a simple but great improvement.
We'll give you some feedback after today's planning meeting.

**Reply by livio-a** [reply]
same as in #2221, good input for reporting feature
